theme.json

```diff
@@ -250,29 +250,34 @@
 				{
 					"fluid": {
 						"min": "0.875rem",
-						"max": "1rem"
+						"max": "3rem"
```

Comment:
@mikachan Wow thank you for these! Question for you or anyone really: Since this is my first foray into this, would I just click 'commit suggestion > commit changes' and then click 'resolve conversation'?

Reply:
Yes, that's right, you can just click 'commit suggestion > commit changes'. That's why I made suggestions instead of just commenting, as it makes it slightly easier for you to make the suggested changes (if you agree with them!)

Alternatively, you can make the changes manually in your code, and then commit and push them as you normally would, if you prefer. These comments will then show as 'outdated', as the code has been updated.